### DISCUSSION TITLE
Undefined bit rate

### DISCUSSION THREAD

**Discussion by readonlymem0ry** [schreibfaul1/ESP32-audioI2S#763] [Q&A]
How do I play audio that doesn't have a defined bit rate?

**Comment by kaloprojects** [comment]
@rixierepo if i remember correctly (i tested once weeks ago) the _'audio.connecttoFS(SD, "/test.wav");'_ example also works if there is no wav header at all (or wav header without bit rate info), i might be wrong, but maybe you should test.
Same with the _'audio.connecttohost("... url../.mp3")'_ streaming (not requesting haeder with bit rate details anyhow).
